plugins/hls-explicit-imports-plugin/src/Ide/Plugin/ExplicitImports.hs

```diff
@@ -175,6 +176,13 @@ instance Show MinimalImportsResult where show _ = "<minimalImportsResult>"
 
 instance NFData MinimalImportsResult where rnf = rwhnf
 
+exportedModuleStrings :: ParsedModule -> [String]
+exportedModuleStrings ParsedModule{pm_parsed_source = L _ HsModule{..}}
+  | Just export <- hsmodExports,
+    exports <- unLoc export
+    = map show exports
```

Comment:
I'm not sure how to extract export module names

Reply:
Does it works as is and you would like to improve it?

Reply:
@jneira Yes. It passed the test I wrote and I can verify it not suggest "explicit import" for exported modules. But i'm not sure it's a good way to implement this by just using `show`. maybe  there is more appropriate function to extract module name I think. :thinking:

Reply:
You could use pattern matching to precisely extract module reexports, but I'm not sure that that's an improvement overall since it also means a stronger coupling to the ghc api, which is bad since we need to support multiple GHC versions. So using `show` seems fine to me.

Reply:
@takoeight0821 I see. thank you for your comment 😄